### ISSUE TITLE
Sync not working

### ISSUE THREAD

**Issue by natrius** [zorgluf/souvenirs-android#11]
Hey, i created an Album and activated the Nextcloud-Sync afterwards. Synced (Dragging down) and no change. I then created an Album afterwards because i thought "Maybe it doesn't sync backwards" but that did not change anything as well. Killed the app and restarted it as well - no change. 

Android 10
Souvenirs: 2.3.2 (F-Droid)

**Comment by zorgluf** [comment]
Syncing can be done afterwards, but is not automatic : you have to click on the "nextcloud" icon on each album you want to sync and everytime you want to sync.
(yes, it's not very intuitive, I am working on that...)

**Comment by natrius** [comment]
There is no Nextcloud button next to the album on the android app. On the other hand, whne dragging down i get the loading-circle - but nothing happens. Am i missing something?

**Comment by zorgluf** [comment]
Well, it's more an image than a button, that's why it's not very intuitive. The image color depends on the sync status of the album
![souvenirs_sync](https://user-images.githubusercontent.com/5617279/160925218-139fca29-ccd8-47fc-b7f8-157517055a3c.jpg)
.

**Comment by natrius** [comment]
Do i need to do something so the symbol appears? Because its not here. I created 2 albums on android. I created a new one and still no symbol. 

But i think i found it. In the settings "Enable Nextcloud Sync" was enabled and "Enable Nextcloud Sync only on Wifi" as well. The problem - i never was on a wifi while testing. Deactivating did not change it as well. But, once i connected to my homewifi the symbol appears suddenly. And even when deactivating wifi-only-sync the symbol now stays until i completely restart the app. 

So, the problem is:
If i change the setting for wifi-only sync the symbol changes just after i restart the app. 

After figuring this out i was able to sync one album finally :D
